base/util.jl

```diff
@@ -251,9 +251,10 @@ unsafe_securezero!(p::Ptr{Cvoid}, len::Integer=1) = Ptr{Cvoid}(unsafe_securezero
 Display a message and wait for the user to input a secret, returning an `IO`
 object containing the secret.
 
-Note that on Windows, the secret might be displayed as it is typed; see
-`Base.winprompt` for securely retrieving username/password pairs from a
-graphical interface.
+!!! info "Windows OS"
```

Comment:
I think it is fine to leave out "OS" here and have people infer that one is talking about the OS from context.

Reply:
Leave out "OS" or "Windows OS" as whole? Kinda confuse at what you mean there.